spec/system/support_interface/view_vendor_api_requests_spec.rb

```diff
@@ -69,8 +69,8 @@ def when_i_visit_the_vendor_api_requests_page
 
   def then_i_see_the_api_request
     expect(page).to have_content('/api/v1/applications/9999/offer')
```

Comment:
Do we want to update this one as well?
```suggestion
    expect(page).to have_selector('p.govuk-body', exact_text: '/api/v1/applications/9999/offer')
```

Reply:
Oops yes missed that will update :+1:

Reply:
@AgaDufrat updated https://github.com/DFE-Digital/apply-for-teacher-training/pull/4315/commits/d279affce4b41ef82f16f4aaa89d3d7e8da29ca9